inSociety/Auth & Sign in/AuthViewController.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  ViewController.swift
+//  inSociety
+//
+//  Created by Aleksey Libin on 12.10.2022.
+//
+
+import UIKit
+import FirebaseAuth
+import FirebaseCore
+import GoogleSignIn
+
+class AuthViewController: UIViewController {
+    
+    private let logoImage = UIImageView(named: "inSociety", contentMode: .scaleAspectFit)
+    private let googleLabel = UILabel(text: "Get started with")
+    private let emailLabel = UILabel(text: "Or sign up with")
+    private let loginLabel = UILabel(text: "Already on board?")
+    
+    private let googleButton = UIButton(title: "Google", titleColor: .black, backgroundColor: .white)
+    private let emailButton = UIButton(title: "email", titleColor: .black, backgroundColor: .mainYellow())
+    private let loginButton = UIButton(title: "Login", titleColor: .mainYellow(), backgroundColor: nil)
+    
+    private let signUpVC = SignUpViewController()
+    private let loginVC = LoginViewController()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupViews()
+        
+        emailButton.addTarget(self, action: #selector(emailButtonPressed), for: .touchUpInside)
+        loginButton.addTarget(self, action: #selector(loginButtonPressed), for: .touchUpInside)
+        googleButton.addTarget(self, action: #selector(googleButtonPressed), for: .touchUpInside)
+        
+        signUpVC.delegate = self
+        loginVC.delegate = self
+        
+    }
+    
+}
+
+
+
+//MARK: - Buttons realization
+extension AuthViewController {
```

Comment:
its better to use as "private extension" if all the methods should be private

Reply:
Resolved